src/bindings/python/src/pyopenvino/graph/model.cpp

```diff
@@ -701,9 +731,87 @@ void regclass_graph_Model(py::module m) {
         return "<" + class_name + ": '" + self.get_friendly_name() + "'\ninputs[\n" + inputs_str + "\n]\noutputs[\n" +
                outputs_str + "\n]>";
     });
+    model.def("get_rt_info",
+              (PyRTMap & (ov::Model::*)()) & ov::Model::get_rt_info,
+              py::return_value_policy::reference_internal,
+              R"(
+                Returns PyRTMap which is a dictionary of user defined runtime info.
+
+                :return: A dictionary of user defined data.
+                :rtype: openvino.runtime.RTMap
+             )");
+    model.def(
+        "get_rt_info",
+        [](const ov::Model& self, const py::list& args) -> py::object {
+            const size_t args_len = py::len(args);
+            std::vector<std::string> cpp_args;
+            for (size_t i = 0; i < args_len; i++) {
+                cpp_args.emplace_back(args[i].cast<std::string>());
+            }
+            return Common::utils::from_ov_any(
+                ov::get_rt_info(self, self.get_rt_info(), cpp_args.cbegin(), cpp_args.cend()));
+        },
+        py::arg("args") = py::list(),
+        R"(
+                Returns runtime attribute.
+
+                :param args: List of strings which defines a path for rt info
+                :type args: List[str]
+
+                :return: A py::object
+             )");
+    model.def(
+        "has_rt_info",
+        [](const ov::Model& self, const py::list& args) -> bool {
+            const size_t args_len = py::len(args);
+            std::vector<std::string> cpp_args;
+            for (size_t i = 0; i < args_len; i++) {
+                cpp_args.emplace_back(args[i].cast<std::string>());
+            }
+            try {
+                ov::get_rt_info(self, self.get_rt_info(), cpp_args.cbegin(), cpp_args.cend());
+            } catch (const ov::Exception&) {
+                return false;
+            }
+            return true;
+        },
+        py::arg("args") = py::list(),
+        R"(
+                Returns true if path exists
+
+                :param args: List of strings which defines a path for rt info
+                :type args: List[str]
+
+                :return: boolean value
```

Comment:
```suggestion
                Checks if given path exists in runtime info of the model.

                :param args: List of strings which defines a path to runtime info.
                :type args: List[str]
                :return: `True` if path exists, otherwise `False`.
                :rtype: bool
```
Small changes to keep Sphinx generator happy. Sounds better right? Please correct my understanding of runtime info if it is wrong. (Similar changes to other docs would be great)